### DISCUSSION TITLE
How can i stop video play in backgroud?

### DISCUSSION THREAD

**Discussion by greenhaixia** [daohoangson/flutter_widget_from_html#643] [Q&A] [answered]
How can I stop video play in background?

**Comment by greenhaixia** [comment]
Youtube link.

**Comment by daohoangson** [comment, marked as answer]
This has been discussed [here](https://github.com/daohoangson/flutter_widget_from_html/issues/37) and [here](https://github.com/flutter/flutter/issues/32363). Basically it's the default behavior of web views in Android.

We do have a parameter to dispose the webview when Android activity is paused, you can use it like this:

```dart
          HtmlWidget(
            html,
            unsupportedWebViewWorkaroundForIssue37: true,
            webView: true,
          ),
```
